docs/advanced-chart-features/advanced-annotations.md

```diff
@@ -0,0 +1,135 @@
+
+### Time cycles
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/time-cycles allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/time-cycles) to check the code.
+
+### Pitchfork
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/pitchfork allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/pitchfork) to check the code.
+
+### Measure
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/measure allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/measure) to check the code.
+
+### Infinity line
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/infinity-line allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/infinity-line) to check the code.
+
+### Fibonacci
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/fibonacci allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/fibonacci) to check the code.
+
+### Fibonacci time zones
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/fibonacci-time-zones allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/fibonacci-time-zones) to check the code.
+
+### Elliot wave
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/elliott-wave allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/elliott-wave) to check the code.
+
+### Crooked line
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/crooked-line allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/crooked-line) to check the code.
+
+### Controllable image
```

Comment:
This and the demo below can be removed - those are not advanced technical annotations, but basic shapes